paddle/pybind/pybind.cc

```diff
@@ -497,6 +505,7 @@ All parameter, weight, gradient are variables in Paddle.
   m.def("init_gflags", InitGflags);
 
   m.def("is_compile_gpu", IsCompileGPU);
+  m.def("get_cuda_device_count", GetCUDADeviceCount());
```

Comment:
Not familiar with Pybind, why it's a function invocation here? `GetCUDADeviceCount()`, rather than `GetCUDADeviceCount`?

Reply:
It is a bug. Thanks for pointing it out.